code/game/machinery/doors/poddoor.dm

```diff
@@ -14,6 +14,12 @@
 	damage_deflection = 70
 	var/id_tag = 1.0
 	var/protected = 1
+	/// Animated icon that is used when the shutter is being opened / closed.
+	var/anim_opening = "opening"
+	var/anim_closing = "closing"
```

Comment:
```suggestion
	// Animated icon that is used when the shutter is being opened / closed.
	var/anim_opening = "opening"
	var/anim_closing = "closing"
```
Either comment them both or remove the autodoc-style comment. Autodoc comments for vars apply to one var, not multiple, so someone could easily confuse both animations for the same var